design_0/test_assert_and_convert.py

```diff
@@ -0,0 +1,5 @@
+from hyp1 import assert_and_convert
+
+def test_assert_and_convert():
```

Comment:
needs a function docstring. usually these provide information on what you're testing